HW_2/generate_password.py

```diff
@@ -0,0 +1,22 @@
+import random
+import string
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/generate-password")
+def generate_password():
+    password_length = random.randint(10, 20)
+
+    return "".join(
+        random.choices(
+            string.digits + string.ascii_letters + string.punctuation, k = password_length
```

Comment:
Your password should always include at least one uppercase letter, one punctuation mark, and one digit. Although your password sometimes doesn't include these elements, this is requirements